--- a/conf/mu.codewith.editor.desktop
+++ b/conf/mu.codewith.editor.desktop
@@ -4,10 +4,10 @@
 GenericName=A Python editor for beginner programmers 
 Comment=A Python editor for beginner programmers
 Icon=mu.codewith.editor
-TryExec=mu
-Exec=mu %F
+TryExec=mu-editor
+Exec=mu-editor %F
 Terminal=false
 Categories=Application;Development;
 Keywords=Python;Editor;microbit;micro:bit;
-StartupWMClass=mu
+StartupWMClass=mu-editor
 MimeType=text/x-python3;text/x-python3;
