--- a/setup.py
+++ b/setup.py
@@ -19,10 +19,10 @@
     license='GPL3',
     packages=['mu', 'mu.contrib', 'mu.resources', 'mu.modes', 'mu.debugger',
               'mu.interface', 'mu.modes.api', ],
-    install_requires=['pycodestyle==2.3.1', 'pyflakes==1.6.0',
-                      'pyserial==3.4', 'pyqt5==5.10', 'qscintilla==2.10.2',
-                      'qtconsole==4.3.1', 'matplotlib==2.1.2',
-                      'pgzero==1.1', 'PyQtChart==5.10'],
+    install_requires=['pycodestyle', 'pyflakes',
+                      'pyserial', 'pyqt5', 'qscintilla',
+                      'qtconsole', 'matplotlib',
+                      'pgzero'],
     include_package_data=True,
     zip_safe=False,
     classifiers=[
