--- a/setup.py
+++ b/setup.py
@@ -1,5 +1,4 @@
 import platform
-import sys
 from setuptools import setup
 from mu import __version__
 
@@ -31,13 +30,6 @@
     # Something unexpected happened, so simply keep all requires
     pass
 
-if not hasattr(sys, 'version_info') or sys.version_info < (3, 6):
-    raise SystemExit(
-        'Mu only works with Python version 3.6 or above. '
-        'For more information see: '
-        'https://codewith.mu/en/howto/install_with_python'
-    )
-
 
 setup(
     name='mu-editor',
